components/index/LoginForm.tsx

```diff
@@ -70,9 +76,21 @@ const LoginForm = () => {
         ) : null}
       </FormGroup>
       <FormBtnContainer>
-        <Button purple={true} onClick={formik.handleSubmit} type="submit">
+        <Button
+          purple={true}
+          onClick={formik.handleSubmit}
+          type="submit"
+          disabled={isLoading ? true : false}
+        >
           Login
         </Button>
+        <SpinnerContainer isLoading={isLoading}>
```

Comment:
make a shared component for the spinner man.

Reply:
fixed